vite.config.ts

```diff
@@ -202,6 +202,15 @@ export default defineConfig(({mode}) => {
 							url: '/teams',
 						},
 					],
+					share_target: {
+						action: '/tasks/create',
+						enctype: 'application/x-www-form-urlencoded',
```

Comment:
This has no effect as we now have a GET route.

Reply:
Yes it does. This is the main thing this PR was opened: to add the share_target to PWA manifest. This describes to the PWA how the share intent must be made.

Reply:
But `application/x-www-form-urlencoded` only works for POST requests. Does it work without this?

Reply:
According to [this](https://developer.chrome.com/en/articles/web-share-target/#accepting-application-changes):
> For the "GET" method, enctype defaults to "application/x-www-form-urlencoded" and is ignored if it's set to anything else.

Stating the enctype here is technically a bit pedantic now that I read that it is the default and only allowed value for GET requests, but yes this is the intended type for GETs. 
(POST requests require `multipart/form-data`)

Reply:
Okay, that makes sense. Thanks for clarifying.